NumLib/DOF/GlobalMatrixProviders.h

```diff
@@ -14,17 +14,14 @@
 
 namespace MathLib
 {
-
-template<typename Vector>
 struct GlobalVectorProvider
 {
-    static VectorProvider<Vector>& provider;
+    static VectorProvider<GlobalVector>& provider;
```

Comment:
Not following all the ideas here, just a question: Why is GlobalVector template argument needed? Couldn't the VectorProvider just use the global type of vectors?